hack/import-restrictions.yaml

```diff
@@ -94,6 +94,7 @@
   - k8s.io/kube-openapi
   - k8s.io/utils
   - k8s.io/klog
+  - k8s.io/component-base
```

Comment:
nit: add this in alphabetical order (I realize klog violate this, but anyway...). For all of these.